asciidoc-confluence-publisher-cli/src/main/java/org/sahli/asciidoc/confluence/publisher/cli/AsciidocConfluencePublisherCommandLineClient.java

```diff
@@ -1,129 +1,130 @@
-        String suffix = argumentsParser.optionalArgument("pageTitleSuffix", args).orElse(null);
-        Map<String, Object> attributes = argumentsParser.optionalJsonArgument("attributes", args).orElseGet(Collections::emptyMap);
-
-        try {
-            AsciidocPagesStructureProvider asciidocPagesStructureProvider = new FolderBasedAsciidocPagesStructureProvider(documentationRootFolder, sourceEncoding);
-            PageTitlePostProcessor pageTitlePostProcessor = new PrefixAndSuffixPageTitlePostProcessor(prefix, suffix);
-
-            AsciidocConfluenceConverter asciidocConfluenceConverter = new AsciidocConfluenceConverter(spaceKey, ancestorId);
-            ConfluencePublisherMetadata confluencePublisherMetadata = asciidocConfluenceConverter.convert(asciidocPagesStructureProvider, pageTitlePostProcessor, buildFolder, attributes);
-
-            ConfluenceRestClient confluenceClient = new ConfluenceRestClient(rootConfluenceUrl, username, password);
-            ConfluencePublisher confluencePublisher = new ConfluencePublisher(confluencePublisherMetadata, publishingStrategy,
-                    confluenceClient, new SystemOutLoggingConfluencePublisherListener(), versionMessage);
-            confluencePublisher.publish();
-        } finally {
-            deleteDirectory(buildFolder);
-        }
-    }
-
-    private static void deleteDirectory(Path buildFolder) throws IOException {
-        walkFileTree(buildFolder, new SimpleFileVisitor<Path>() {
-
-            @Override
-            public FileVisitResult visitFile(Path path, BasicFileAttributes attributes) throws IOException {
-                delete(path);
-
-                return CONTINUE;
-            }
-
-            @Override
-            public FileVisitResult postVisitDirectory(Path path, IOException e) throws IOException {
-                delete(path);
-
-                return CONTINUE;
-            }
-
-        });
-    }
-
-
-    private static class SystemOutLoggingConfluencePublisherListener implements ConfluencePublisherListener {
-
-        @Override
-        public void pageAdded(ConfluencePage addedPage) {
-            System.out.println("Added page '" + addedPage.getTitle() + "' (id " + addedPage.getContentId() + ")");
-        }
-
-        @Override
-        public void pageUpdated(ConfluencePage existingPage, ConfluencePage updatedPage) {
-            System.out.println("Updated page '" + updatedPage.getTitle() + "' (id " + updatedPage.getContentId() + ", version " + existingPage.getVersion() + " -> " + updatedPage.getVersion() + ")");
-        }
-
-        @Override
-        public void pageDeleted(ConfluencePage deletedPage) {
-            System.out.println("Deleted page '" + deletedPage.getTitle() + "' (id " + deletedPage.getContentId() + ")");
-        }
-
-        @Override
-        public void publishCompleted() {
-            System.out.println("Documentation successfully published to Confluence");
-        }
-
-    }
-
-}
+/*

```

Comment:
It looks like you might have changed the line endings from `LF` to `CRLF` - that is why all lines are marked as changed. Could you please convert it back to `LF` only?